config/environments/test.rb

```diff
@@ -37,6 +37,10 @@
   # Inline ActiveJob when testing:
   config.active_job.queue_adapter = :inline
 
+  # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
+  # the I18n.default_locale when a translation can not be found)
+  config.i18n.fallbacks = true
```

Comment:
Is there a benefit to not enabling fallbacks for the dev environment as well? (That's the only place it won't be enabled once this is merged.)

Reply:
Not enabling fallbacks means we get errors when translation keys are missing. So that doesn't add much because:

- We plan to do https://otwarchive.atlassian.net/browse/AO3-6003 and have automated checks for missing keys.
- I was worried that `it_behaves_like "a translated email"` would become useless, but even with fallbacks, such assertions would still fail if we're running in the default locale and have no keys for "en" (there's nothing else to fall back on).

Should I enable fallbacks in config/application.rb for all environments?

Reply:
Oh lol:

https://github.com/otwcode/otwarchive/blob/6b99402ac54eba0b220e85163408c777322c2251/config/environments/development.rb#L31-L33

config/application.rb it is!

Reply:
... y'know, I am getting _quite_ tired of being failed by my text editor's multi-file search. 😒